js/src/logger.ts

```diff
@@ -2348,6 +2348,47 @@ export function traced<IsAsyncFlush extends boolean = false, R = void>(
   }
 }
 
+export function trace<
```

Comment:
We should probably have documentation for this function, similar to the `@traced` decorator.

Reply:
Added.